.github/CODEOWNERS

```
@@ -0,0 +1 @@
+@frrist
```

is this the same as this?
```suggestion
*	@frrist
```

---

:shrug: github says this current file is valid? So I assume it's fine until more folks wanna join as owners?